modules/security-context-constraints-psa-synchronization.adoc

```diff
@@ -9,11 +9,20 @@
 
 {product-title} includes link:https://kubernetes.io/docs/concepts/security/pod-security-admission[Kubernetes pod security admission]. Globally, the `privileged` profile is enforced, and the `restricted` profile is used for warnings and audits.
 
+[NOTE]
+====
+If you want to preview _enforcing_ the `restricted` profile on your non-production cluster as a Technology Preview, you can enable the `TechPreviewNoUpgrade` feature set. If you enable the `TechPreviewNoUpgrade` feature set, pods are rejected if they violate pod security standards, instead of only logging a warning.
```

Comment:
```suggestion
If you want to preview _enforcing_ the `restricted` profile on your non-production cluster as a Technology Preview, you can enable the `TechPreviewNoUpgrade` feature set. If you enable the restricted profile, pods are rejected if they violate pod security standards, instead of only logging a warning.
```

Reply:
Updated to:

> If you enable the `TechPreviewNoUpgrade` feature set, which enables the `restricted` profile, pods ...

To make it clear that by enabling the feature set, the restricted profile is enabled (you don't have to do anything other than enable the TP feature set to enable the restricted profile)